### DISCUSSION TITLE
Smart Plug unable to Pair

### DISCUSSION THREAD

**Discussion by unintended-consequences** [Koenkk/zigbee2mqtt#17376] [Devices]
I have two units of a smart plug. I am unable to pair them with z2m. Any troubleshooting suggestions?

- They are likely sourced from aliexpress but not 100% sure as I have ordered many smart plugs and jumbled them up
- They are marked Model: RMC003, Wireless type: ZigBEE 2.4GHz
- When I hold the button on the side for 5 seconds, I can get it to flash (appears to be in pairing mode) but they will not join my network. No error codes to google :(

Z2M is functioning perfectly, 1.30.3, I have paired 20+ other devices today, including 3 other types/brands of smart plugs. Coordinator is zStack3x0 running on a smartlight.me CC2562P stick

**Comment by hownioni** [comment]
did you ever make it work?
